src/rpc/net.cpp

```diff
@@ -749,9 +749,10 @@ static RPCHelpMan listbanned()
             {
                 {RPCResult::Type::OBJ, "", "",
                     {
-                        {RPCResult::Type::STR, "address", ""},
-                        {RPCResult::Type::NUM_TIME, "banned_until", ""},
-                        {RPCResult::Type::NUM_TIME, "ban_created", ""},
+                        {RPCResult::Type::STR, "address", "The IP/Subnet of the banned node"},
+                        {RPCResult::Type::NUM_TIME, "banned_until", "The " + UNIX_EPOCH_TIME + " the node is banned until"},
+                        {RPCResult::Type::NUM_TIME, "ban_created", "The " + UNIX_EPOCH_TIME + " the ban was created"},
+                        {RPCResult::Type::NUM_TIME, "ban_time", "The amount of time the ban will last for in seconds"},
```

Comment:
```suggestion
                        {RPCResult::Type::NUM_TIME, "ban_time", "The ban duration, in seconds"},
```

Reply:
Thanks for making my original text look like a rube goldberg machine 🤗

Reply:
addressed in 8e34b5ed06f3ca0865b9b7dca6e31f2e95c6f5c8